module/apmgopg/hook.go

```diff
@@ -0,0 +1,76 @@
+// ownership. Elasticsearch B.V. licenses this file to you under
```

Comment:
Looks like the copyright header is missing bits at the front. This tripped up CI:

    module/apmgopg/hook.go: is missing the license header

Reply:
Fixed and squashed everything into a single commit